HeterogeneousCore/SonicTriton/src/TritonClient.cc

```diff
@@ -336,10 +339,14 @@ inference::ModelStatistics TritonClient::getServerSideStatus() const {
 
 //for fillDescriptions
 void TritonClient::fillPSetDescription(edm::ParameterSetDescription& iDesc) {
+  edm::ParameterSetDescription descConverter;
+  fillBasePSetDescription(descConverter);
+  descConverter.add<std::string>("converterName");
   edm::ParameterSetDescription descClient;
   fillBasePSetDescription(descClient);
   descClient.add<std::string>("modelName");
   descClient.add<std::string>("modelVersion", "");
+  descClient.add<edm::ParameterSetDescription>("converterDefinition", descConverter);

```

Comment:
I keep thinking of this and forgetting to comment on it: a single converter definition per client isn't sufficient for the general case. Triton supports networks with multiple (named) inputs and outputs that can have different types. This will probably require two VPSets, `inputConverters` and `outputConverters` to be fully general, with each PSet in these VPSets having both the input/output name and the converter name.

Reply:
Ok, good point. I finally have FaaST updated to v2 so that I can test properly, so I should be able to get this change included soon.